scripts/case_change_service_test.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for scripts/case_change_service.py."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.tests import test_utils
+
+import python_utils
+
+from . import case_change_service 
+
+class CheckCaseChangeServiceTests(test_utils.GenericTestBase):
```

Comment:
Please add more tests to make sure you cover all python primitives.

Reply:
Done

Reply:
How about a set?

Reply:
This service is used while changing the case of the json response. I don’t think we pass sets in json response.

Reply:
Your function may be used in other cases, so it's not limited to the specific use case in this PR.

Reply:
Ok, done!